dashboard/app/controllers/scripts_controller.rb

```diff
@@ -48,15 +46,11 @@ def edit
   end
 
   def update
-    respond_to do |format|
-      script_text = params[:script_text]
-      if @script.update_text(script_params, script_text, i18n_params)
-        format.html { redirect_to @script, notice: I18n.t('crud.updated', model: Script.model_name.human) }
-        format.json { head :no_content }
-      else
-        format.html { render action: 'edit' }
-        format.json { render json: @script.errors, status: :unprocessable_entity }
```

Comment:
Ah sorry, try the diff [ignoring whitespace](https://github.com/code-dot-org/code-dot-org/pull/10942/files?w=1).  I removed the JSON format response (which we don't use) but kept the redirect to 'edit' on save failure.